src/adera/Machines/Rocket.cpp

```diff
@@ -173,11 +175,13 @@ void SysMachineRocket::update_physics(ActiveScene& rScene)
         Vector3 torque = Magnum::Math::cross(location, thrust);
         Vector3 worldTorque = rCompTf.m_transform.transformVector(torque);
         SysPhysics_t::body_apply_torque(rCompRb, worldTorque);
- 
+        
+        rCompRb.m_rigidbodyDirty = true;
```

Comment:
The rocket changed the inertia?

Reply:
In the current configuration, the rocket is the system that tells the containers how much fuel is being withdrawn from the container. I know you didn't like this before, but until we have a better pipe/wire system I'm not sure if there's a reasonable way to resolve it. In any case, it makes sense to set the dirty flag wherever the fuel withdrawal is calculated.